dali/operators/reader/loader/coco_loader.h

```diff
@@ -59,6 +63,39 @@ inline bool HasSavePreprocessedAnnotationsDir(const OpSpec &spec) {
     (spec.HasArgument("dump_meta_files_path") && spec.GetArgument<bool>("dump_meta_files_path"));
 }
 
+struct RLEMask {
```

Comment:
I wonder... if you expose all the fields for modification, then this looks more like a C++ variant of the RLE structure - then maybe this should be handled by inheritance?
```suggestion
struct RLEMask : RLE {
```
Otherwise, you can treat RLE as a resource and use `struct RLEMask : UniqueResource<RLE>`:
```C++
struct RLEMask : UniqueResource<RLE> {

  RLEMask(const char* str, int h, int w) {
    rleFrString(&handle_, const_cast<char*>(str), h, w);
  }
  RLEMask(span<const unsigned int> counts, int h, int w) {
    rleInit(&handle_, h, w, counts.size(), const_cast<unsigned int*>(counts.data()));
  }

  void DestroyHandle(RLE rle) {
     if (rle.cnts)
       rleFree(&rle);
  }
};
```

Reply:
done